internal/app/pimo/play.go

```diff
@@ -135,6 +140,36 @@ func checkSecurityRequirements(pdef model.Definition) error {
 	return nil
 }
 
+func checkMask(mask model.MaskType) error {
```

Comment:
It's the responsibility of the mask to know if it's secure or not. Each time a new mask is added we have to update this function. Maybe we could append a `isSecure()` function on mask interface. and filter on that attribute.

Reply:
If we do that, we will filter `template` `add` and `add-transient`  masks, which are very important masks ..

We must know if a mask is secure at execution with parameters (eg: a template mask is insecure only if it contains a call to an insecure mask..)